Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/FieldsHelper.cs

```diff
@@ -67,61 +67,61 @@ public static int CreateField(IRSAPIClient client, FieldRequest request)
 			//Create the field
-			var resultSet = client.Repositories.Field.Create(fieldDTO);
+			DTOs.WriteResultSet<DTOs.Field> resultSet = client.Repositories.Field.Create(fieldDTO);
 			//Check for success
 			if (resultSet.Success)
 			{
 				fieldID = resultSet.Results.FirstOrDefault().Artifact.ArtifactID;
 			}
 			else
 			{
-				var innEx = resultSet.Results.Any() ? new Exception(resultSet.Results.First().Message) : null;
+				Exception innEx = resultSet.Results.Any() ? new Exception(resultSet.Results.First().Message) : null;
 				throw new Exception(resultSet.Message, innEx);
 			}
 			return fieldID;
 		}
 		public static int CreateField_Date(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.Date);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.Date);
 			return CreateField(client, fieldRequest);
 		}
 		public static int CreateField_User(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.User);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.User);
 			return CreateField(client, fieldRequest);
 		}
 		public static int CreateField_FixedLengthText(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.FixedLengthText);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.FixedLengthText);
 			return CreateField(client, fieldRequest);
 		}
 		public static int CreateField_LongText(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.LongText);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.LongText);
 			return CreateField(client, fieldRequest);
 		}
 		public static int CreateField_WholeNumber(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.WholeNumber);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.WholeNumber);
 			return CreateField(client, fieldRequest);
 		}
 		public static int CreateField_YesNO(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.YesNo);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.YesNo);
 			return CreateField(client, fieldRequest);
 		}
 		public static int CreateField_SingleChoice(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.SingleChoice);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.SingleChoice);
 			return CreateField(client, fieldRequest);
 		}
 		public static int CreateField_MultipleChoice(IRSAPIClient client, int workspaceID)
 		{
-			var fieldRequest = new FieldRequest(workspaceID, FieldType.MultipleChoice);
+			FieldRequest fieldRequest = new FieldRequest(workspaceID, FieldType.MultipleChoice);
 			return CreateField(client, fieldRequest);
```

Comment:
Is this returning an artifacr id? If the old method was returning an artifact id, make sure we do that here as well

Reply:
Yeah its the same thing, just changed var to be the explicit type FieldRequest

Reply:
thanks